### PR TITLE
[NEWSTACK-105] PSO 6.2.0-rc1 release for NVMeoF-RoCE support

### PR DESCRIPTION
**Pull request by pure-jliao**

### CODE REVIEW

**Review by sdodsley — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v6.2.0-rc1 (07/22/2021)
+
+#### Features:
+
+* FlashArray NVMeoF-RoCE support (NVMeoF using RDMA over Converged Ethernet)
+* A fresh PSO installation is recommended to use this release
```
Comment (dsupure) [inline CHANGELOG.md:8]:
```suggestion
* A fresh PSO installation is recommended to use this release only if you want to try the NVMeoF-RoCE feature.
```

File: docs/flasharray-nvmeof.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# FlashArray NVMeoF Support
+
+## Introduction
+
+The Pure Service Orchestrator Kubernetes CSI driver includes support for FlashArray NVMeoF RDMA SAN type
+starting from PSO v6.2.0, explicitly NVMeoF-RoCE.
+
+## Prerequisites
```
Comment (dsupure) [inline docs/flasharray-nvmeof.md:8]: Please add the prerequisites in the readme.

File: docs/flasharray-nvmeof.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+* PSO v6.2.0 or higher
+* FlashArray Purity 5.2.0 or higher
+* nvme-cli 1.8.1 or higher
+* Mellanox OFED 2.3 or higher
```
Comment (dsupure) [inline docs/flasharray-nvmeof.md:13]: Do we need `Mellanox` requirement?  Do NVMe team has a doc to specify the requirement for FA-NVMe-RoCE? We can add the reference here.

**Review by dsupure — COMMENTED** [review]
leave some comments to address. Look good in general.

**Review by david-enli — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v6.2.0-rc1 (07/22/2021)
+
+#### Features:
+
+* FlashArray NVMeoF-RoCE support (NVMeoF using RDMA over Converged Ethernet)
+* A fresh PSO installation is recommended to use this release only if you want to try the NVMeoF-RoCE feature.
```
Comment (Pure-AdamuKaapan) [inline CHANGELOG.md:8]: I would flip this sentence order, and then upgrade it to a "must" (I don't think I'd trust moving a cluster from iSCSI or FC to NVMe in place).
```suggestion
* If you want to try the NVMeoF-RoCE feature, a fresh PSO installation is required.
```

File: pure-pso/README.md [hunk 5]
```diff
@@ -199,7 +199,7 @@ The following table lists the configurable parameters and their default values.
...
 | `flasharray.iSCSIAllowedCIDR`                  | List of CIDR blocks allowed as iSCSI targets, e.g. 10.0.0.0/24,10.1.0.0/16. Use comma (,) as the separator, and empty string means allowing all addresses. | ""                                            |
 | `flasharray.preemptAttachments`                | Enable/Disable attachment preemption!                                                                                                                      | `true`                                        |
-| `flasharray.sanType`                           | Block volume access protocol, either ISCSI or FC                                                                                                           | `ISCSI`                                       |
+| `flasharray.sanType`                           | Block volume access protocol, ISCSI, FC or NVMEOF-RDMA                                                                                                     | `ISCSI`                                       |
```
Comment (Pure-AdamuKaapan) [inline pure-pso/README.md:202]:
```suggestion
| `flasharray.sanType`                           | Block volume access protocol, one of `ISCSI`, `FC`, or `NVMEOF-RDMA`                                                                                                     | `ISCSI`                                       |
```

File: docs/flasharray-nvmeof.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## Usage
+
+### Before starting
+
+* Follow our [knowledge base](https://support.purestorage.com/Solutions/Linux/Procedures/NVMe//RoCE_Initiator_Setup_for_RHEL//CentOS_7.6)
+to setup and configure your hosts to use NVMe first (skip persistent settings). PSO doesn't configure the environment automatically.
+* If you are running a PSO version lower than 6.2.0, a fresh installation (or a reinstall followed by DB reconstruction) 
+is recommended as there are database schema changes in 6.2.0 release.
+* PSO only support one FlashArray SAN type so you cannot mix NVMe with other SAN types.
```
Comment (Pure-AdamuKaapan) [inline docs/flasharray-nvmeof.md:25]:
```suggestion
* PSO only supports using one FlashArray SAN type at a time. Mixing SAN types (for example, NVMe and iSCSI in the same cluster) is not supported.
```

**Review by Pure-AdamuKaapan — COMMENTED** [review]
Looking pretty good!

**Pull request merged**